src/Extension/SiteConfigExtension.php

```diff
@@ -91,7 +86,7 @@ protected function getHelpLink()
         return sprintf(
             '<a class="d-block mfa-settings__help-link" target="blank" rel="noopener" href="%s">%s</a>',
             $link,
-            _t(__CLASS__ . '.FIND_OUT_MORE', 'Find out more')
+            _t(__CLASS__ . '.FIND_OUT_MORE', 'Learn about MFA')
```

Comment:
This has a different purpose to the translation key now, maybe rename the key as well